plugins/package-managers/conan/src/main/kotlin/PackageInfo.kt

```diff
@@ -30,10 +30,35 @@ private val JSON = Json {
     namingStrategy = JsonNamingStrategy.SnakeCase
 }
 
-internal fun parsePackageInfos(file: File): List<PackageInfo> = JSON.decodeFromString(file.readText())
+internal fun parsePackageInfosV1(file: File): List<PackageInfo> =
+    JSON.decodeFromString<List<PackageInfoRaw>>(file.readText()).map { it.toPackageInfo() }
+
+/**
+ * A class containing the properties common to all [PackageInfo], regardless of their Conan version. Used for
+ * abstracting some functions and keeping them in [Conan].
+ */
+@Serializable
+sealed interface Info {
```

Comment:
IMO it would be clearer if this interface was named `PackageInfo`, and `PackageInfo` became `PackageInfoV1`.